test.sh

```diff
@@ -50,6 +50,5 @@ end pytest
 
 start docker
 ./docker.sh 5001
-server_up 5001
```

Comment:
Did you determine this check to be redundant?

Reply:
Yeah: Cypress has its own retry until it's up, so we don't need our own reinvented wheel here.